### PR TITLE
VACMS-16146 Add VBA banner cypress test.

### PR DESCRIPTION
**Pull request by swirtSJW**
## Description

Relates to #16146

Test can be run locally with
```
ddev composer va:test:cypress -- --spec=tests/cypress/integration/features/content_type/facilities/vba/vba_facility.feature
```

**Note:** if this test fails mid-test, it is possible that subsequent runs will fail because the node content is locked by the first fake test user.

## QA steps

What needs to be checked to prove this works?
What needs to be checked to prove it didn't break any related things?
What variations of circumstances (users, actions, values) need to be checked?

Since this is a testing only issue, the passing of the tests themselves are what should be QA'd
1. In the demo environment, 
   - [x] validate that the test ran and passed.
   
image

2. Look at the test in the PR
   - [ ] Validate that it covers banner related field visibility (when they should show, and when they should not.
   - [ ] Validate that if the node is saved, the values of the banner fields are not saved

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.
- [ ] If there are field changes, front end output has been thoroughly checked.

### Select Team for PR review

- [ ] `CMS Team`
- [ ] `Public websites`
- [x] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Comment by swirtSJW** [comment]
Bizarre.   Test is failing but the PR says all tests are passing.
image

File: tests/cypress/integration/weights.json [hunk 1]
```diff
@@ -1 +1 @@
...on/features/platform/file_upload.feature":{"time":68310,"weight":8},"tests/cypress/integration/features/platform/generate_automatic_url_alias.feature":{"time":64351,"weight":8},"tests/cypress/integration/features/platform/help_center.feature":{"time":37695,"weight":4},"tests/cypress/integration/features/platform/image_crop.feature":{"time":94,"weight":0},"tests/cypress/integration/features/platform/media.feature":{"time":310540,"weight":40},"tests/cypress/integration/features/platform/navigation.feature":{"time":16790,"weight":2},"tests/cypress/integration/features/platform/password_policy.feature":{"time":35899,"weight":4},"tests/cypress/integration/features/platform/permissions.feature":{"time":84030,"weight":10},"tests/cypress/integration/features/platform/text_validation.feature":{"time":104535,"weight":13},"tests/cypress/integration/features/platform/tooltip.feature":{"time":8767,"weight":1},"tests/cypress/integration/features/platform/translation.feature":{"time":128853,"weight":16},"tests/cypress/integration/features/platform/user_login.feature":{"time":43462,"weight":5},"tests/cypress/integration/features/taxonomy_type/va_benefit_taxonomy.feature":{"time":36221,"weight":4}}
```
Comment (swirtSJW) [inline tests/cypress/integration/weights.json:1]: This was only edited to change the paths to the facilities related features since they were moved.

**Review by omahane — APPROVED** [review]
Thanks for organizing these better. The tests passed and read well. No questions or concerns.

**Comment by swirtSJW** [comment]
image

**Pull request merged**